Add minder technical charter.

riaankleinhans opened the pull request:
# Summary

Add Minder Technical Charter

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/mindersec/minder?pullRequest=4834) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/mindersec/minder?pullRequest=4834) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/mindersec/minder?pullRequest=4834) it.</sub>

puerco commented:
Hey @riaankleinhans thanks for adding this, I have a couple of questions:

- Do you have a markdown version? I think I would like to check in the first version as test to track the evolution of the document via its commit history.
- Do we need to host the charter on the main site? In other projects, I've hosted the charter in the community repository (for [example in openvex](https://github.com/openvex/community/blob/main/GOVERNANCE.md)), which helps keep the governance documents neatly separate from the code. Of course, we would point back to it from here and the other repositories.

riaankleinhans commented:
@puerco the LF Project formation team prefer that the PDF version is kept on in the repo.
There is not suppose to be any changes to the document without the approval  of the LF Formation team.
You can convert it to .md without change, as it is a legal document.
The document can live in any of folders within this repo and should be easily discoverable by the community.

evankanderson commented:
See https://github.com/mindersec/community/pull/11 -- I've converted to markdown and updated the document we'd checked in ahead of time in the `community` repository.  There are also a few changes from what we proposed that I'd love to get @riaankleinhans feedback on.

riaankleinhans commented:
> See [mindersec/community#11](https://github.com/mindersec/community/pull/11) -- I've converted to markdown and updated the document we'd checked in ahead of time in the `community` repository. There are also a few changes in the PDF from what we proposed that I'd love to get @riaankleinhans feedback on.

@evankanderson moving the document to the community repo would not be a problem as it is still within the GH Org dedicated to the project.
However the changes proposed in the mindersec/community#11 can not be made without the LF Formation team updating the PDF document.
Due to he fact that a change that seem like a "cosmetic" change to the average contributor could have a serious legal impact on the project, the Formation team require that the document must be added to the repo without any changes.

If the changes are important to the project, I can request for it to be reviewed and updated and a new PDF be issued.

evankanderson commented:
The changes in https://github.com/mindersec/community/pull/11 *undo* changes that we'd wanted to make, to make the checked-in document match the LF PDF here.

I'd checked in our red-lined copy of the document when creating that repository, before the LF legal review was completed. 

I'm happy to go through the appropriate process to get the changes we care about that differ from the PDF approved or countered by LF legal (probably mostly the electronic vote simplification).  The most important part for me was getting our existing copy corrected, and making sure that future changes were tracked and guarded effectively.

riaankleinhans commented:
@evankanderson @puerco 
 Closed in favor of https://github.com/mindersec/community/pull/11
